bindings/rascal/models/IP_generic_md.py

```diff
@@ -0,0 +1,103 @@
+import ase.io
+import ase.units
```

Comment:
I'm not fond of the coupling with ASE here, but I understand we are already using it for structures loading. In order to reduce such coupling, could we use a different solution for unit management?

Reply:
Units are already handled by i-PI, so this import is now unused. I'll take care of it with the rest of the changes.